### PR TITLE
Control panel

### PR DESCRIPTION
**Pull request by mf222**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
   }).$mount('selector');
-})
-
-
 
+  document.getElementById('panel').appendChild(document.createElement('panel'));
+  const panel = new Vue({
+    render: h => h(ConfigPanel)
+  }).$mount('panel');
+})
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Missing semicolon                              semi

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+  const selector = new Vue({
+    render: h => h(Selector)
   }).$mount('selector');
-})
-
-
 
+  document.getElementById('panel').appendChild(document.createElement('panel'));
+  const panel = new Vue({
+    render: h => h(ConfigPanel)
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Missing trailing comma                         comma-dangle

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
-    render: h => h(App)
+  const selector = new Vue({
+    render: h => h(Selector)
   }).$mount('selector');
-})
-
-
 
+  document.getElementById('panel').appendChild(document.createElement('panel'));
+  const panel = new Vue({
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: 'panel' is assigned a value but never used     no-unused-vars

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
-import App from '../app.vue';
+import Selector from '../app.vue';
+import ConfigPanel from '../panel.vue';
 
 document.addEventListener('DOMContentLoaded', () => {
   document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App)
+  const selector = new Vue({
+    render: h => h(Selector)
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: Missing trailing comma                         comma-dangle

File: app/javascript/packs/application.js [hunk 5]
```diff
@@ -1,15 +1,18 @@
...
 import Vue from 'vue';
-import App from '../app.vue';
+import Selector from '../app.vue';
+import ConfigPanel from '../panel.vue';
 
 document.addEventListener('DOMContentLoaded', () => {
   document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App)
+  const selector = new Vue({
```
Comment (monkeyci) [inline app/javascript/packs/application.js]: 'selector' is assigned a value but never used  no-unused-vars

File: app/assets/stylesheets/base/_global.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+$secondaryBorder: #f0f0f0;
+$primaryBorder: #f1f1f1;
+$primary: #49475e;
+$white: #fff;
+$black: #000;
+$orange: #ff9e00;
+$gray: #dedede;
+$text: #aeabcc;
+$modalBackground: rgba(0, 0, 0, .5);
+$modalShadow: rgba(0, 0, 0, .33);
```
Comment (monkeyci) [inline app/assets/stylesheets/base/_global.scss]: Unexpected missing end-of-source newline   no-missing-end-of-source-newline

File: app/assets/stylesheets/app/modal.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+.modal-leave-active {
+  opacity: 0;
+}
+
+.modal-enter .modal-container,
+.modal-leave-active .modal-container {
+  -webkit-transform: scale(1.1);
+  transform: scale(1.1);
+}
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Unexpected missing end-of-source newline       no-missing-end-of-source-newline

File: app/assets/stylesheets/app/modal.scss [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  opacity: 0;
+}
+
+.modal-leave-active {
+  opacity: 0;
+}
+
+.modal-enter .modal-container,
+.modal-leave-active .modal-container {
+  -webkit-transform: scale(1.1);
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Unexpected vendor-prefix "-webkit-transform"   property-no-vendor-prefix

File: app/assets/stylesheets/app/modal.scss [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  margin: 0px auto;
+  padding: 20px 30px;
+  background-color: $white;
+  border-radius: 2px;
+  box-shadow: 0 2px 8px $modalShadow;
+  transition: all .3s ease;
+  font-family: $roboto;
+}
+
+.modal__header__title{
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Expected single space before "{"               block-opening-brace-space-before

File: app/assets/stylesheets/app/modal.scss [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+  display: table;
+  transition: opacity .3s ease;
+}
+.modal__wrapper {
+  display: table-cell;
+  vertical-align: middle;
+}
+.modal-container {
+  width: 300px;
+  margin: 0px auto;
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Unexpected unit                                length-zero-no-unit

**Comment by mf222** [comment]
[Panel de Configuración](https://trello.com/c/3QTAan8I/48-panel-de-configuraci%C3%B3n)

File: app/assets/stylesheets/app/modal.scss [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+  height: 100%;
+  background-color: $modalBackground;
+  display: table;
+  transition: opacity .3s ease;
+}
+.modal__wrapper {
+  display: table-cell;
+  vertical-align: middle;
+}
+.modal-container {
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Expected empty line before rule                rule-empty-line-before

File: app/assets/stylesheets/app/modal.scss [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+  z-index: 9998;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: $modalBackground;
+  display: table;
+  transition: opacity .3s ease;
+}
+.modal__wrapper {
```
Comment (monkeyci) [inline app/assets/stylesheets/app/modal.scss]: Expected empty line before rule                rule-empty-line-before

**Pull request closed without merging**